### PR TITLE
Remove RNG hit chance at close range

### PR DESCRIPTION
**Pull request by ValoTheValo**
## About The Pull Request
Kills RNGesus, Removes the accuracy debuff you get at close range for some reason.  why was this even added?
(curse you github for not letting me embed images) https://cdn.discordapp.com/attachments/534648497861951488/1057496131455680573/TacticalCombat.png

## Why It's Good For The Game
having a random miss chance at close range generally was counterintuitive and is annoying to deal with. without it combat should feel less loose and random and more tight and skillbased

## Testing
i compiled then dumped 9 mags into a test dummy

## Changelog
:cl:
del: Removed accuracy debuff at close range
/:cl:

### CODE REVIEW

File: code/modules/projectiles/projectile.dm [hunk 1]
```diff
@@ -288,7 +288,6 @@
 	var/hit_mod = 0
 	switch(target_mob.mob_size)
 		if(120 to INFINITY)
-			hit_mod = -6
```
Comment (Humonitarian) [inline code/modules/projectiles/projectile.dm:291]: What's up with this one?

Comment (ValoTheValo) [reply code/modules/projectiles/projectile.dm:291]: from what i could tell that was what was generating the random missses. im not really a coder so i may be wrong

Comment (Humonitarian) [reply code/modules/projectiles/projectile.dm:291]: this one increases the hit chance if a mob is bigger than Iriska (megafauna and mechs)

Comment (ValoTheValo) [reply code/modules/projectiles/projectile.dm:291]: > this one increases the hit chance if a mob is bigger than Iriska (megafauna and mechs)

so a negative hit mod indicates decreased chance to hit. wierd

Comment (ValoTheValo) [reply code/modules/projectiles/projectile.dm]: reverted that part

**Review by Humonitarian — APPROVED** [review]

**Review by hyperioo — COMMENTED** [review]
> Removes the accuracy debuff you get at close range for some reason. why was this even added?

I guess the idea was that when something is close to you, if it moves one meter, it's a big change from where you are aiming. If it's 10 meters away, then it barely moves and you don't have to adjust your aim that much.

**Comment by ValoTheValo** [comment]
> > Removes the accuracy debuff you get at close range for some reason. why was this even added?
> 
> I guess the idea was that when something is close to you, if it moves one meter, it's a big change from where you are aiming. If it's 10 meters away, then it barely moves and you don't have to adjust your aim that much.

Still just makes combat generally more annoying to do.

**Review by hyperioo — APPROVED** [review]

**Pull request merged**